pymc3/tuning/starting.py

```diff
@@ -171,3 +176,126 @@ def allinmodel(vars, model):
     notin = [v for v in vars if v not in model.vars]
     if notin:
         raise ValueError("Some variables not in the model: " + str(notin))
+
+
+
+class Monitor(object):
```

Comment:
that sounds like a reasonable test -- somewhere in `test_starting.py`.  It'd be super tricky to test that it actually does what is intended, but just running the code will make sure it keeps up with API changes.